Lib/test/test_capi.py

```diff
@@ -999,6 +999,30 @@ async def foo(arg): return await arg  # Py 3.5
             self.assertEqual(ret, 0)
             self.assertEqual(pickle.load(f), {'a': '123x', 'b': '123'})
 
+    def test_int_max_str_digits_config_is_per_interp(self):
```

Comment:
Would you mind to add a short comment explaining the purpose of the test? I understand that it checks that each interpreter has its own limit, and changing the limit of sub-interpreter doesn't affect the main interpreter.